src/pages/Login.tsx

```diff
@@ -0,0 +1,48 @@
+import LoginWithKakao from '../components/user/LoginWithKakao';
+import styled from 'styled-components';
+import { authState } from '../store/recoil';
+import { useSetRecoilState } from 'recoil';
+import { useNavigate } from 'react-router-dom';
+
+const Login = () => {
+    const setAuth = useSetRecoilState(authState);
+    const navigate = useNavigate();
+    return (
+        <LoginPage>
+            <LoginWithKakao />
+            <LoginWithGuest
+                onClick={() => {
+                    setAuth({ userName: '게스트', profileImage: 'https://i.ibb.co/kBV7WsL/2022-05-05-10-58-23.png', init: true });
+                    navigate('/');
+                }}
+            >
+                귀찮아요, 그냥 들어가게 해주세요
+            </LoginWithGuest>
+        </LoginPage>
+    );
+};
+
+export default Login;
+
+const LoginPage = styled.div`
+    display: flex;
+    height: 100%;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    background-color: #fcec5c;
+    @media screen and (min-width: 600px) {
+        border-radius: 15px;
+    }
```

Comment:
```suggestion
    height: 100%;

    display: flex;
    justify-content: center;
    align-items: center;
    flex-direction: column;

    background-color: #fcec5c;

    @media screen and (min-width: 600px) {
        border-radius: 15px;
    }
```
관련 있는 CSS 코드를 분리하면 가독성에 도움이 되는것 같습니다!
(코딩 스타일 영역이라 단지 추천 입니다!)